tests/sourcemapping/public/decipher_poll_dapp.py

```diff
@@ -0,0 +1,207 @@
+# This example is provided for informational purposes only and has not been audited for security.
```

Comment:
Temporary placement. I can add it to a better location. @jasonpaulos : should I put it under `examples/applications/abi` ?